99.md

```diff
@@ -0,0 +1,170 @@
+# NIP-99
+
+## Prediction Markets
+
+`draft` `optional` `author:ekzyis`
+
+This NIP specifies how prediction markets can be implemented using nostr and lightning.
+
+Any user ("the creator") can create a _predictable event_ (`kind 1099`) such that other users ("the participants") can buy and sell shares of possible outcomes using zap requests as defined by NIP-57 and reverse zap requests (TBD).
+Services ("the betting exchanges") the required events for buying and selling shares, set the prices per share (based on odds) and handle payouts.
+Participants can freely choose in which exchange(s) they want to participate, e.g. based on liquidity or reputation since participants must trust the exchanges with payouts and reporting the honest outcome of the event.
+Exchanges can charge commissions on winning bets to incentivize continuous honest behaviour.
+
+For a general overview on what prediction markets are, see Appendix A.
+
+### Creation of predictable events
+
+To create a predictable event, `kind 1099` events are used as defined by NIP-01 like this:
+
+```json
+{
+  "content": <description about what should be predicted>,
+  "tags": [
+    ["outcome": <description of possible outcome>],
+    ["outcome": <description of another possible outcome>],
+    ...
+  ],
+  "kind": 1099,
+  "id": <event_id>
+  "pubkey": <pubkey of creator>,
+  "created_at": <timestamp>,
+  "sig": <signature>
+}
+```
+
+Example for a binary option event:
+
+```json
+{
+  "content": "Will Twitter go bankrupt in 2023?",
+  "tags": [
+    ["outcome": "Yes"],
+    ["outcome": "No"]
+  ],
+  "kind": 1099,
+  ...
+}
+```
+
+Exchanges run clients which have subscribed to these events.
+Per received event, they create parametrized replaceable events as defined by NIP-33 per possible outcome:
+
+<!--
+Working on a better explanation. Exchanges don't have to provide services for every event.
+
+Also, I want to provide an example flow from start to finish.
+This example flow should especially answer the following questions regarding "market bootstrapping":
+  Q: How are initial prices set?
+  A: Without any participants, there is no information regarding odds so every outcome is equally probable. So every share should cost the same.
+
+  Q: What happens if there will only be one participant? Do they not always lose money because of potential commission fees?
+  A: ???
+
+  Q: Why would the first participant buy more than a single share? If they buy multiple shares, the next participant has an advantage because the next participant can just buy a single share to profit the most with the least risk (assuming no other participant will follow).
+  Basically this: Why would you bet 100 sats to win 100 sats if you can just bet 1 sat to win 100 sats?
+  A: ??? Maybe I am just confused with shares vs. odds. I need to learn more about prediction markets first.
+-->
```

Comment:
I have iterated a few times over this NIP already but I still have some important questions to answer.

I added them as HTML comments to the NIP so I don't forget them